neo/rawio/spikeglxrawio.py

```diff
@@ -408,8 +408,12 @@ def extract_stream_info(meta_file, meta):
             # https://github.com/billkarsh/SpikeGLX/blob/15ec8898e17829f9f08c226bf04f46281f106e5f/Markdown/Metadata_30.md#imec
             # We allow also LF streams for NP2.0 because CatGT can produce them
             # See: https://github.com/SpikeInterface/spikeinterface/issues/1949
-            per_channel_gain[:-1] = 1 / 80.
-            gain_factor = float(meta['imAiRangeMax']) / 8192
+            if 'imChan0apGain' in meta:
+                per_channel_gain[:-1] = 1 / float(meta['imChan0apGain'])
```

Comment:
For NP2, the LF stream can be produced only by CatGT (see https://github.com/SpikeInterface/spikeinterface/issues/1949), so I believe it's ok to use the AP gain in that case too

Reply:
since these changes are releated to the SpikeGLX version, should they be propaated to NP1 probes too? @guptadivyansh what do you think?

Reply:
The per channel gain part is already read from the imro table for 1.0, so that should be correct as is.

I didn't touch the immaxInt (512) for 1.0, because I assumed all 1.0 probe were 10bit and didn't want to introduce changes that might make people's old data inconsistent with new runs of Neo. Although it seems that immaxInt has been in SpikeGLX for a while. So we could also change it for 1.0, for consistency with 2.0 and only use the hardcoded 512 if the immaxInt is not in Meta.

If 1.0 were all 10bit, and I would rather not change it. But if you think we should, I can update the PR later this week.

Reply:
Makes sense. I think that all NP1.0 are 10bit so no need to update ;)